src/ezcord/bot.py

```diff
@@ -54,18 +54,30 @@ def __init__(
         elif error_webhook_url:
             self.logger.warning("You need to enable error_handler for the webhook to work.")
 
-    def load_cogs(self, directory: str = "cogs"):
+    def load_cogs(self, directory: str = "cogs", subdirectories: bool = False):
         """Load all cogs in a given directory.
 
         Parameters
         ----------
         directory: :class:`str`
             Name of the directory to load cogs from.
             Defaults to ``cogs``.
+        subdirectories: :class:`bool`
+            Whether to load cogs from subdirectories.
+            Defaults to ``False``.
         """
-        for filename in os.listdir(f"./{directory}"):
-            if filename.endswith(".py"):
-                self.load_extension(f'{directory}.{filename[:-3]}')
+        if subdirectories is False:
+            for filename in os.listdir(f"./{directory}"):
+                if filename.endswith(".py"):
+                    self.load_extension(f'{directory}.{filename[:-3]}')
+
+        elif subdirectories is True:
+            for path, subs, files in os.walk(f"./{directory}"):
+                for name in files:
+                    if name.endswith(".py"):
+                        p_str = path.replace("\\", ".")
+                        p_str = p_str.replace("/", ".")
+                        self.load_extension(f'{p_str}.{name[:-3]}')
```

Comment:
Oh oh, hast mal wieder verkackt richtigen Python code zu schreiben 🤨. Hast du code von Ryzm kopiert???
Mach es mal lieber so:

```suggestion
def load_cogs(self, directory: str = "cogs", sub_dirs: bool = False) -> None:
    """Load all cogs in a given directory.
    Parameters
    ----------
    directory: :class:`str`
        Name of the directory to load cogs from.
        Defaults to ``cogs``.
    subdirectories: :class:`bool`
        Whether to load cogs from subdirectories.
        Defaults to ``False``.
    """
    if sub_dirs:
        for file in os.scandir(directory):
            if file.name.endswith(".py"):
                self.load_extension(f"{directory}.{file.name[:-3]}")

    else:
        for dir in os.scandir(directory):
            if dir.is_dir():
                for sub_file in os.scandir(dir.path):
                    if sub_file.name.endswith(".py"):
                        self.load_extension(f"{directory}.{dir.name}.{sub_file.name[:-3]}")
```